openlibrary/components/BulkSearch/components/BookCard.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+
+<div class="book-card">
+          <div class="cover">
+            <img :src="coverImage">
+          </div>
+          <div class="info">
+            <a class="title" :href="workUrl" target="_blank">{{ doc.title }}</a>
+            <div class="byline">{{ doc.author_name[0] }}</div>
+
+            <div class="identifier">{{doc.first_publish_year}} - {{doc.edition_count}} editions</div>
+            <a class="action actionName" target="_blank"></a>
+          </div>
+</div>
+
+
+</template>
+
+
+
+<script>
+
+export default {
+    props: {
+        doc: Object
+    },
+    computed: {
+        coverImage() {
+            if (!this.doc.cover_i){
+                return ''
+            }
+            return `https://covers.openlibrary.org/b/id/${this.doc.cover_i}-M.jpg`
+        },
+        workUrl() {
+            return `https://openlibrary.org/books/${this.doc.key.split('/')[2]}`
```

Comment:
Books and works are interchangeable, so far as I recall.  I think you're right though, this could probably be `https://openlibrary.org/${this.doc.key}`, from what I'm looking at.